include/irr/video/CDerivativeMapCreator.h

```diff
@@ -1,3 +1,7 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
```

Comment:
@Crisspl we can delete this file now... yes?

Reply:
TODO

Reply:
yeah, basically yes. But dont forget to remove all references from engine. CDerivativeMapCreator object is still created somewhere if I recall correctly...

Reply:
@Przemog TODO